waspc/test/AnalyzerTest.hs

```diff
@@ -190,6 +196,18 @@ spec_Analyzer = do
             ]
       takeDecls @Action <$> decls `shouldBe` Right expectedAction
 
+      let expectedJob =
+            [ ( "BackgroundJob",
+                Job.Job
+                  { Job.perform =
+                      ExtImport
+                        (ExtImportField "backgroundJob")
+                        (fromJust $ SP.parseRelFileP "jobs/baz.js")
+                  }
+              )
+            ]
+      takeDecls @Job <$> decls `shouldBe` Right expectedJob
```

Comment:
I think you can safely remove `@Job` (same goes for most of the older tests).

Reply:
Thanks, will give it a try!

Reply:
I was able to remove all type applications except this one, which the compiler could not automatically infer:

```haskell
takeDecls @Route <$> analyze source
  `shouldBe` Left (TypeError $ TC.mkTypeError (ctx (1, 34) (1, 48)) $ TC.UndefinedIdentifier "NonExistentPage")
```

Thanks! 👍🏻